packages/docs/content/utilities/font/font-weight.mdx

```diff
@@ -6,18 +6,14 @@ import loremIpsumGenerator from '../../../src/helpers/loremIpsumGenerator';
 
 Change an element's `font-weight` property using the font weight utility.
 
-**Format**: `.ds-u-font-weight--[normal|semibold|bold]`
+**Format**: `.ds-u-font-weight--[normal|bold]`
 
 <EmbeddedExample>
   <section class="example-list">
     <article>
       <code class="ds-u-margin-bottom--1">.ds-u-font-weight--normal</code>
       <div class="ds-u-font-weight--normal">{loremIpsumGenerator('m')}</div>
     </article>
-    <article>
-      <code class="ds-u-margin-bottom--1">.ds-u-font-weight--semibold</code>
```

Comment:
Did we wanna add this to our debug CSS to make sure nobody's using it in the wild?

Reply:
I ripped it right out, but I guess it wouldn't hurt to put a deprecation notice on it as well.